.github/workflows/update-translations.yml

```diff
@@ -0,0 +1,38 @@
+on: workflow_dispatch
+
+name: Update transifex translations
+jobs:
+  update_translations:
+    name: Update transifex translations
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install transifex client
+        run: |
+          mkdir -p /home/runner/.local/bin
+          cd /home/runner/.local/bin
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          sudo apt-get install -y gettext
+      - name: Run update-translations script
+        # From https://github.com/marketplace/actions/create-pr-action:
+        # > executes an arbitrary command and commits the changes to the new pull request
+        uses: technote-space/create-pr-action@v2
+        env:
+          TRANSIFEX_TOKEN: ${{ secrets.TRANSIFEX_TOKEN }}
+          UPDATE_TRANSLATIONS_SKIP_GIT: 1
+        with:
+          EXECUTE_COMMANDS: |
+            git submodule update --init --recursive > /dev/null 2>&1
+            pip install --upgrade pip > /dev/null 2>&1
+            # install requirements, excluding difficult to install ones
+            grep -Ev 'xmlsec|python3-saml' requirements/dev-requirements.txt > ../requirements.txt
+            pip install -r ../requirements.txt  > /dev/null 2>&1
+            # copy .transifexrc.example to ~/.transifexrc, edited to include token
+            grep -v '^token =' .transifexrc.example > ~/.transifexrc
+            echo "token = ${TRANSIFEX_TOKEN}" >> ~/.transifexrc
+            ./scripts/update-translations.sh
+          COMMIT_MESSAGE: |
+            Update translations.
+
+            [ci skip]
+          PR_BRANCH_NAME: 'update-translations'
```

Comment:
What happens if there is an existing branch with this name?

Reply:
That's a good question. The actual branch name ends up being `create-pr-action/update-translations` so it's unlikely to conflict with a branch a developer is working on. But I'm not sure how it would handle multiple uses on different runs of the same branch name. Looking at the docs, all the examples include `-${PR_ID}` at the end of the `PR_BRANCH_NAME` value, so I'm going to follow that example and avoid having to worry about how it handles that case. Thanks!

Reply:
Actually I think I'm going to do this in a followup PR. Not sure if `$PR_ID` is the _new_ PR or the one PR that the action is running on (which isn't relevant to us since we'll be running on master), but it probably is the latter, in which case what I said here isn't helpful to us.

Reply:
Wow, @millerdev  you were right to to raise this: it does the dumbest possible thing: it just uses whatever's on that branch already (https://github.com/dimagi/commcare-hq/pull/31438). Figuring out how to deal with this...